projects/js-toolkit/packages/liferay-cli/src/new/facet-angular/index.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+import {
+	FilePath,
+	TRANSFORM_OPERATIONS,
+	TemplateRenderer,
+	format,
+	transformJsonFile,
+	transformTextFile,
+} from '@liferay/js-toolkit-core';
+
+import ensureOutputFile from '../util/ensureOutputFile';
+
+import type {Facet, Options} from '../index';
+
+const {
+	PkgJson: {addDependencies},
+	Text: {appendLines},
+} = TRANSFORM_OPERATIONS;
+const {info, print} = format;
+
+const facet: Facet = {
+	async prompt(useDefaults: boolean, options: Options): Promise<Options> {
+		return options;
+	},
+
+	async render(options: Options): Promise<void> {
+		print(info`Generating sample code...`);
+
+		const renderer = new TemplateRenderer(
+			new FilePath(__dirname).join('templates'),
+			options.outputPath
+		);
+
+		const pkgJsonFile = ensureOutputFile(options, 'package.json');
+		/* eslint-disable-next-line @typescript-eslint/no-var-requires, @liferay/no-dynamic-require */
+		const pkgJson = require(pkgJsonFile.asNative);
+
+		const context = {
+			...options,
+			pkgJson,
+		};
+
+		print(info`  Configuring TypeScript compiler`);
+
+		await renderer.render('tsconfig.json', context);
+
+		// TODO: add build step: tsc
```

Comment:
Oops. I will remove this in the next PR (for Vue). It is already done in this PR, in the target platforms commit :+1: